src/config_store.c

```diff
@@ -92,9 +108,56 @@ void ConfigStore_Init(ConfigStore *p)
     p->_fd = -1;
 }
 
+/*To delete the leftover tmp files on the device startup*/
+void DeleteFileHelper(struct dirent *file, char *filePath)
```

Comment:
Mark it `static` since it's a internal helper that is not part of the library's interface.

Reply:
Function is marked as static.